README.recovery

```diff
@@ -17,7 +17,7 @@ The process event is responsible for executing synchronization.  This event
 will return a state as to whether it has completed or not.  This allows for
 synchronization to be interrupted and recontinue when the message queue buffer
```

Comment:
I'm not sure exactly what is meant by _recontinue_. If there's a suggestion, I can reword it in this PR.